example/windows/NetInfoExample/Package.appxmanifest

```diff
@@ -45,5 +45,6 @@
 
   <Capabilities>
     <Capability Name="internetClient" />
+    <DeviceCapability Name="wiFiControl"/>

```

Comment:
Can you add this requirement as a note in the README? Also, is it backwards compatible? Stated differently: if this device capability is *not* in appxmanifest, does it fall back to the previous unknown string(s) ?

Reply:
Should be fixed now. Let me know if there are any other changes needed.